Index: function.fetch.php
===================================================================
RCS file: /cvsroot/tikiwiki/tiki/Smarty/plugins/function.fetch.php,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- function.fetch.php	8 Oct 2002 13:54:47 -0000	1.1
+++ function.fetch.php	22 Dec 2002 08:23:44 -0000	1.2
@@ -17,7 +17,7 @@
         return;
     }
 
-    if ($smarty->security && !preg_match('!^(http|ftp)://!i', $file)) {
+    if ($smarty->security && !preg_match('!^(http|https|ftp)://!i', $file)) {
         // fetching file, make sure it comes from secure directory
         foreach ($smarty->secure_dir as $curr_dir) {
             if (substr(realpath($file), 0, strlen(realpath($curr_dir))) == realpath($curr_dir)) {
@@ -41,7 +41,7 @@
 		}
     } else {
 		// not a local file
-		if(preg_match('!^http://!i',$file)) {
+		if(preg_match('!^https?://!i',$file)) {
 			// http fetch
 			if($uri_parts = parse_url($file)) {
 				// set defaults
